src/com/videojs/providers/HLSProvider.as

```diff
@@ -231,6 +252,45 @@ package com.videojs.providers{
           }
         }
 
+        /**
+         * This is called whenever a segment is successfully loaded.
+         *
+         * Whenever a segment is loaded we keep metrics on a series of statistics, such as
+         * - how many segments have been requested
+         * - how many seconds of media have been downloaded
+         * - how many bytes have been transferred
+         * - how much times has been spent downloading segments
+         * - what the bandwidth was for the latest downloaded segment
+         */
+        private function _onFragmentLoaded(event:HLSEvent):void
+        {
+          var metrics = event.loadMetrics;
+          var fragmentTransferTime = metrics.parsing_end_time - metrics.loading_request_time;
```

Comment:
You could use `metrics. processing_duration` here https://github.com/brightcove/flashls/blob/dev-brightcove/src/org/mangui/hls/event/HLSLoadMetrics.as#L38